dandi/tests/test_metadata.py

```diff
@@ -234,6 +235,12 @@ def test_timedelta2duration(td: timedelta, duration: str) -> None:
     assert timedelta2duration(td) == duration
 
 
+@pytest.mark.xfail(
+    condition="not config.getoption('--scheduled')",
+    reason="Flaky ontobee site",
+    strict=False,
+    raises=requests.RequestException,
+)
```

Comment:
could you please place this into `@mark.xfail_ontobee` similarly to `@mark.skipif_no_network` to avoid duplication/clutter?

Reply:
@jwodder ping on this one so we could bring our tests back into greenland

Reply:
@yarikoptic The custom `mark` machinery only seems to support custom `skipif` decorators, not `xfail`.  I think creating a custom `mark.xfail_ontobee` would be more trouble than it's worth.

Reply:
Hmm, ok

What about just binding decorator into a helper "variable" eg `@mark_xfail_ontobee` or that wouldn't work?

Reply:
It seems to work, but now the tests are failing due to #1424.

Reply:
ok, since seems to be unrelated -- let's proceed. Thank you @jwodder